core/context.go

```diff
@@ -286,14 +301,27 @@ func (ctx *Context) UpdateBackend(vsID, rsID string, weight int32) (int32, error
 	// Save the old backend weight and update the current backend weight.
 	result, rs.options.Weight = rs.options.Weight, weight
 
+	// Currently the backend options are changing only the weight.
```

Comment:
I'm not sure what's the best approach here – let's table it for now and leave it as it is done in this PR currently.